chorus_lib/examples/hello.rs

```diff
@@ -41,7 +43,10 @@ fn main() {
     let mut handles: Vec<thread::JoinHandle<()>> = Vec::new();
     // Create a transport channel
     // let transport_channel = LocalTransportChannel::<LocationSet!(Bob, Alice)>::new();
-    let transport_channel = LocalTransportChannel::new().with(Alice).with(Bob);
+    let transport_channel = LocalTransportChannelBuilder::new()
+        .with(Alice)
+        .with(Bob)
+        .build();
     // Run the choreography in two threads
     {
         // let transport_channel = transport_channel.clone();
```

Comment:
also this

Reply:
Fixed 1a0f0e1da56163f169f03775398da0dba254b2a7